docs/040_pepr-tutorials/030_create-pepr-operator.md

```diff
@@ -130,7 +130,9 @@ export class WebApp extends a.GenericKind {
 }
 ```
 
-in the `source` folder, create a file called `webapp.crd.ts` and add the following:
+Move the updated file to `capabilities/crd/generated/webapp-v1alpha1.ts`.
+
+in the `capabilities/crd/source` folder, create a file called `webapp.crd.ts` and add the following, we want the controller to automatically create the CRD when it starts.
```

Comment:
```suggestion
In the `capabilities/crd/source` folder, create a file called `webapp.crd.ts` and add the following. This will have the controller automatically create the CRD when it starts.
```